src/main/java/seedu/address/model/body/CauseOfDeath.java

```diff
@@ -0,0 +1,9 @@
+package seedu.address.model.body;
+
+/**
+ * Enumerates common causes of death.
+ */
+public enum CauseOfDeath {
```

Comment:
I don't think we need an enum for cause of death since there can be many causes. Can perhaps just declare the `cod` var in `Body` as a String.

Reply:
Noted, Shao Yi. I will make the changes!

Reply:
Done.